src/Components/benchmarkapps/Wasm.Performance/TestApp/Pages/TimerComponent.razor

```diff
@@ -0,0 +1,38 @@
+@page "/timer"
+@inject IJSRuntime JSRuntime
+@using System.Threading
+@implements IDisposable
+
+<h1>Timer component</h1>
+<div style="width: 100px; height: 100px; background-color: rgb(@red, @green, @blue))">
+Hello world
+</div>
+
+@code
+{
+    Random random = new Random();
+    Timer timer;
+    int red = 128;
+    int green = 128;
+    int blue = 128;
+
+    protected override void OnInitialized()
+    {
+        timer = new Timer(UpdateColor, null, 300, 300);
```

Comment:
This seems pretty odd to me. It means we wait 300ms before the first callback, right? If so, doesn't the 300ms massively dominate the whole benchmark timing?

I would have guessed this would make the benchmark results much less useful, as we'll have to compare numbers like 306ms versus 312ms (and get the impression nothing has really changed) versus 6ms vs 12ms (where it's clear there's a big difference).

Why is the delay useful? Isn't the scenario equally valid if there's zero delay before the first callback?

Reply:
I can tweak this to fire off immediately and have the test wait for updated. I don't run this as part of perf, this scenario is primarily meant for stress runs to test out timer callbacks and disposable components. I couldn't think of a proper perf scenario to make out of this - do you have any suggestions?

Reply:
Thanks for clarifying.

Maybe now we need to separate the stress and perf tests, since they have different goals. It doesn't make sense to include this timer component in perf tests, as the results will be kind of useless and kind of misleading.

Reply:
I guess that might be nice, but we would have to do a bunch of infrastructure work to support this - basically the Driver needs to be able to switch between perf vs stress. Do you feel strongly about separating the two?

Reply:
No, I'm fine with reusing the same app if that helps a lot.

However maybe we could prefix the test page names and test classes with something to indicate which one they are for (e.g., `StressOrgChart` vs `PerfOrgChart`).

Reply:
Just in the cases where it's relevant to distinguish them, that is.